packages/@sfcc-bff/basketapi/src/api/schema/basketResolvers.ts

```diff
@@ -103,16 +111,57 @@ const getProductsDetails = async (
     return result.data;
 };
 
+const getFullProductItems = async (
```

Comment:
Created this re-usable function to get all the details of a product in the basket object. Original logic was done in getBasket but I needed it in coupons so turned that logic into this function.